scss/_tables.scss

```diff
@@ -86,11 +86,23 @@
 //
 // Placed here since it has to come after the potential zebra striping
 
+// stylelint-disable selector-max-type
```

Comment:
Please enable the rule again after the block you want to suppress the warning

Reply:
Sorry about that, I have re-enabled the selector-max-type rule after the block.